src/retro_data_structures/formats/pak_wii.py

```diff
@@ -38,10 +48,12 @@ def _emitparse_header(code: construct.CodeGen) -> str:
 ConstructResourceHeader._emitparse = _emitparse_header
 
 PAKNoData = Struct(
-    _start=construct.Tell,
+    _start=construct.Tell,      ##Should always be 0x00
     _header=PAKHeader,
+    _table_of_contents_start=construct.Tell,        ##Should always be 0x40
+    ##Not entirely sure what DictConstruct is supposed to represent
     table_of_contents=construct.Aligned(64, make_dict(Int32ub, FourCC)),
-    _named_resources_start=construct.Tell,
+    _named_resources_start=construct.Tell,      ##Should always be 0x80
```

Comment:
Not necessarily, since the table of contents has a dynamic length.

Sometimes a thing is dynamic, even if every file in the game doesn't have that value vary.